keras_cv/models/efficientnet_v1.py

```diff
@@ -26,7 +26,8 @@
 
 import tensorflow as tf
 from keras import backend
```

Comment:
For consistency, let's also import `keras.backend` using `from tensorflow.keras import backend`

Reply:
Fixed